test/zimcheck-test.cpp

```diff
@@ -87,6 +87,27 @@ class CapturedStdout
   operator std::string() const { return buffer.str(); }
 };
 
+class CapturedStderr
+{
+  std::ostringstream buffer;
+  std::streambuf* const sbuf;
+public:
+  CapturedStderr()
+    : sbuf(std::cerr.rdbuf())
+  {
+    std::cerr.rdbuf(buffer.rdbuf());
+  }
+
+  CapturedStderr(const CapturedStderr&) = delete;
+
+  ~CapturedStderr()
+  {
+    std::cerr.rdbuf(sbuf);
+  }
+
+  operator std::string() const { return buffer.str(); }
+};
+
```

Comment:
It would be better to make `CapturedStdOut` configurable (and rename it to `CatpturedStdStream`) than creating almost the same class.

Reply:
Done, but I preserved `CapturedStdout` and `CapturedStderr` as thin wrappers around `CapturedStdStream` in order to keep intact the 16 locations where they were used. In my opinion,

```
CapturedStdout stdout;
```
is more readable (less noisy) than

```
CapturedStdStream stdout(std::cout);
```